content/content-tool/tool/src/webapp/vm/content/sakai_resources_exportFinish.vm

```diff
@@ -48,8 +45,23 @@
                 </tr>
             #end
         </table>
+		<h3>
```

Comment:
The indentation here looks wrong, could you try to match it with the surrounding context, the issue maybe because of differences tabs and spaces.